parts/k8s/kubernetescustomscript.sh

```diff
@@ -24,12 +24,20 @@ ensureRunCommandCompleted()
 {
     echo "waiting for runcmd to finish"
     wait_for_file 900 1 /opt/azure/containers/runcmd.complete
+    if [ ! -f /opt/azure/containers/runcmd.complete ]; then
+        echo "Timeout waiting for runcmd to complete"
```

Comment:
can you add a mention of cloud-init here? maybe "Timeout waiting for cloud-init runcmd to complete"